integration-tests/src/test/java/org/apache/fineract/integrationtests/AdvancedPaymentAllocationLoanRepaymentScheduleTest.java

```diff
@@ -5473,6 +5475,45 @@ public void uc149() {
         });
     }
 
+    // UC150: Validate interestRateFrequencyType to set as Whole Term at the Loan account level
```

Comment:
Would you enhance this test case or create a new one which validate the proper interest rate is set and used? I would say validating whether the proper annual interest rate is on the loan and the calculated interest is matching would be sufficient

Reply:
Done! Fix to have the right annual interest rate using whole term option applied